src/styles/styles.js

```diff
@@ -648,7 +648,7 @@ const styles = {
     },
 
     chatItemComposeSecondaryRow: {
-        minHeight: 15,
+        height: 15,
```

Comment:
NAB: why?

Reply:
Same as with the other one, have a look at that PR and it was changed to `minHeight` there so changing it back to `height`. Havent tried without.